core/config/config.go

```
@@ -519,13 +522,16 @@ func ConfigureRegion(cfg *Configuration) error {
 		// Region is not available.
 		return fmt.Errorf("the provided region is not available for this API, available regions are: %s", availableRegions)
 	}
-	// Global API. The provided region is ignored.
-	// If the url is a template, generated using deprecated config.json, the region variable is replaced
-	// If the url is already configured, there is no region variable and it remains the same
-	cfgUrl := strings.Replace(servers[0].URL, "{region}", "", -1)
+
+	// Global API. If a region is provided return an error.
+	// The region is not part of the url anymore instead the region is provided via path parameter.
+	if strings.Contains(servers[0].URL, "{region}") {
+		return fmt.Errorf("a region was provided in the url for a global api but has to be specified in the individual methods; STACKIT will move to a new way of specifying regions with v1")
+	}
```

We only want to return an error if the server URL does not contain a `{region}` placeholder

---

And the message can be something simple like "this API does not support setting a region in the server URL, please check if the region can be specified as a function parameter"

---

Are these two comments not contradictory? 
The error message of your second comment fits for if the URL contains a `{region}`.